src/App.js

```diff
@@ -67,23 +55,14 @@ class App extends React.Component {
 
   state = {
     ...INITIAL_DAO_STATE,
-    connected: false,
     fatalError: null,
     identityIntent: null,
-    selectorNetworks: SELECTOR_NETWORKS,
     transactionBag: null,
     signatureBag: null,
     web3: getWeb3(web3Providers.default),
     wrapper: null,
   }
 
-  componentDidMount() {
-    // Only the default, because the app can work without the wallet
-    pollConnectivity([web3Providers.default], connected => {
-      this.setState({ connected })
```

Comment:
As far as I could tell, this `connected` state is actually not used anymore now that we do the block polling in the Account module.

Good riddance—this was spiking our API requests to ETH nodes!

Reply:
🚀